pkg/controller/node/controller_utils.go

```diff
@@ -320,3 +326,11 @@ func terminatePods(kubeClient clientset.Interface, recorder record.EventRecorder
 	}
 	return complete, nextAttempt, nil
 }
+
+func NormalLimiterFunc(nodeNum int, defaultQPS float32) float32 {
```

Comment:
If we are leaving as is, please change the names - it's not limitter - it's about QPS for it.